gravitee-apim-console-webui/src/management/api/notifications/notifications-list/notifications-list.component.ts

```diff
@@ -13,11 +13,81 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Component } from '@angular/core';
+import { Component, Inject, OnInit } from '@angular/core';
+import { takeUntil, tap } from 'rxjs/operators';
+import { Subject } from 'rxjs';
+
+import { NotificationSettingsService } from '../../../../services-ngx/notification-settings.service';
+import { UIRouterStateParams } from '../../../../ajs-upgraded-providers';
+import { GioTableWrapperFilters } from '../../../../shared/components/gio-table-wrapper/gio-table-wrapper.component';
+import { gioTableFilterCollection } from '../../../../shared/components/gio-table-wrapper/gio-table-wrapper.util';
+
+type NotificationSettingsTable = {
+  name: string;
+  id: string;
+};
+
+export interface NotificationSettingsTableApi {
+  id: string;
+  name: string;
+  referenceType: string;
+  referenceId: string;
+  notifier: string;
+  hooks: [];
```

Comment:
```suggestion
  hooks: string[];
```
hooks is a string array